### PR TITLE
DOP-4516: Make unhandled search query log more specific

### PR DESCRIPTION
**Pull request by mayaraman19**
### Ticket

DOP-4516

### Notes

See instructions to test down below. `JSON.stringify()` here printed an empty object, but `String()` prints the error message.

Not sure if I need to update other error logging in this file.

This PR also updates the `search.test.ts` timer to 30s instead of 20s so the test doesn't time out as frequently!

### README updates

- - [ ] This PR introduces changes that should be reflected in the README, and I have made those updates.
- - [X] This PR does not introduce changes that should be reflected in the README

### CODE REVIEW

File: src/Marian/index.ts [hunk 1]
```diff
@@ -116,7 +116,7 @@ export default class Marian {
       results = await this.fetchResults(parsedUrl);
```
[the remark this answers is not in the document]
Comment (seungpark) [reply src/Marian/index.ts:116]: can we test this with a new InvalidQuery error? would also want to address that second A/C of what search query was used

**Review by seungpark — COMMENTED** [review]
left a comment below! please include the original query as part of the first log 🙏

File: src/Marian/index.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (mayaraman19) [reply src/Marian/index.ts:116]: whoops missed that, thanks

File: .mocharc.json [hunk 2]
```diff
@@ -2,5 +2,5 @@
   "extension": ["ts"],
   "spec": ["tests/**/**/**.test.ts"],
   "require": "ts-node/register",
-  "timeout": "20000"
+  "timeout": "30000"
```
Comment (mayaraman19) [inline .mocharc.json:5]: not sure if this should be changed, but some of the failing tests still showed a 20000ms timeout even after changing the other timeout to 30000ms so I figured that it should be changed here as well

Comment (seungpark) [reply .mocharc.json:5]: i believe [this change](https://github.com/mongodb/docs-search-transport/pull/108/files#diff-d83d3d9cf0be5dce464942d69b396844b0731533be16c4b9edcf44875a23a007R44) should only be for [the test suite](https://mochajs.org/#suite-level) but i am fine with bumping this up in general

**Review by seungpark — APPROVED** [review]
LGTM 👍

**Pull request merged**